src/base/flamec/supermatrix/hip/main/FLASH_Queue_hip.c

```diff
@@ -21,6 +21,9 @@
 static FLA_Bool flash_queue_enabled_hip  = FALSE;
 static FLA_Bool flash_malloc_managed_hip = FALSE;
 static dim_t    flash_queue_hip_n_blocks = 128;
+static dim_t    flash_queue_hip_blocksize = 1024;
```

Comment:
no - these (and the get/set) shouldn't go here. they should go into the lapack2flash layer (and they apply to all queues not just the HIP one, so names shouldn't reference HIP)

Reply:
Got it, will address!

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/926e05ee52bc9f34e87d35af34eb71042ab6b6ed